extension/chrome/elements/compose-modules/formatters/encrypted-mail-msg-formatter.ts

```diff
@@ -96,6 +96,16 @@ export class EncryptedMsgMailFormatter extends BaseMailFormatter {
     //    - flowcrypt.com/shared-tenant-fes (consumers and customers without on-prem setup), or
     //    - fes.customer-domain.com (enterprise customers with on-prem setup)
     //    It will be served to recipient through web
+    const { to, cc, bcc } = newMsg.recipients;
+    const updateContactName = (recipient: { email: string; name: string }) => {
+      return {
+        ...recipient,
+        name: Xss.stripEmojis(recipient.name),
+      };
+    };
+    newMsg.recipients.to = to?.map(updateContactName);
+    newMsg.recipients.cc = cc?.map(updateContactName);
+    newMsg.recipients.bcc = bcc?.map(updateContactName);
```

Comment:
I checked code for uploading messages to FES, and we already apply some formatting to recipients before upload: https://github.com/FlowCrypt/flowcrypt-browser/blob/c0cb65990e174951b31464bf71e863bef4640685/extension/js/common/api/account-servers/external-service.ts#L146

Let's move `stripEmojis` there and apply it to formatted email:
```ts
to: (recipients.to || []).map(Str.formatEmailWithOptionalName).map(Xss.stripEmojis),
...
```

Reply:
Thanks - adding .map() that would perform the `Xss.stripEmojis` callback function is a better solution and with less code to write.